x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -502,9 +502,8 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
     (groupingFilters: Filter[]) => {
       return (
         <AlertsTableComponent
-          configId={ALERTS_TABLE_REGISTRY_CONFIG_IDS.RULE_DETAILS}
+          tableType={TableId.alertsOnRuleDetailsPage}
```

Comment:
nit: I am a bit confused with the change of the name here. What is the `tableType` in the new table?

Reply:
They were distinguishing table configurations based on the page/area where the table was used. These cases are called "ids", but are not actually ids in the proper sense (AlertsTable's `id` prop). I kept this logic for now (I'm sure we can work with Security to improve this a lot, perhaps splitting it in multiple components and removing this logic altogether), but I'm using the term "type" to avoid confusing it with proper table ids

Reply:
Makes sense to me.